### PR TITLE
Fix openAI and mindmap issue

### PR DESCRIPTION
**Pull request by dockabernathy01**
Fix openAI and mindmap issues

- Bugfix

### CODE REVIEW

File: src/components/settings/AddonsTab.vue [hunk 1]
```diff
@@ -96,11 +99,14 @@ export default {
...
+      this.config = newValue;
+    },
   },
   computed: {
-    ...mapGetters({
-      config: "config/fullConfig",
-    }),
+    // ...mapGetters({
+    //   config: "config/fullConfig",
+    // }),
```
Comment (dacoaster) [inline src/components/settings/AddonsTab.vue:109]: @dockabernathy01 you reverted this to the "old style" of passing configs.  Please see how all of the other `settings` tabs handle this and store the data in state rather than via props.  These props like `configItem` are exactly the sort of thing that should be getting removed in this issue: https://github.com/yatt-ai/yattie/issues/147

So please revert these changes around props and getters.

Comment (dockabernathy01) [reply src/components/settings/AddonsTab.vue:109]: Ok, got it.

**Comment by dockabernathy01** [comment]
@dacoaster , I have updated the code.
